static/mojang/library-patches.json

```diff
@@ -1617,12 +1617,12 @@
       {
         "downloads": {
           "artifact": {
-            "sha1": "c6606e57db075ad218a2e78d2416c159a53b6a0c",
-            "size": 157996,
-            "url": "https://build.lwjgl.org/release/3.3.1/bin/lwjgl-jemalloc/lwjgl-jemalloc-natives-linux-arm64.jar"
+            "sha1": "d710714af1b06a11fc34ad3d675fbaa28455a7ab",
+            "size": 158917,
+            "url": "https://github.com/theofficialgman/lwjgl3-binaries-arm64/raw/lwjgl-3.3.1/lwjgl-jemalloc-patched-natives-linux-arm64.jar"
```

Comment:
Are you talking about the jar contents? It will still function packed flat. I forgot that with lwjgl 3.2.3+ the folder model was added as an alternative form.

Reply:
yes, here was the commit that did that: https://github.com/LWJGL/lwjgl3/commit/5bafbc71af483b2dda2729e6330ce2eee1731902#diff-05ec5fc63e3939a989ea4746c6184d0e1d2e367bc2433f4ea25ccbb20ad556eb

I'll do some testing locally later today to verify exactly what loading systems function on lwjgl 3.3.1

Reply:
@Scrumplex can confirm, packing flat (binaries in root) as well as the folder structure (linux/arm64/org/lwjgl/jemalloc) both work. Would you for me to repack the jar to use the nested folder structure?